config/sql/migration/migrate.23.sql

```diff
@@ -0,0 +1,12 @@
+-- Don't modify this! Create a new migration instead--see docs/schema-migrations.adoc
+SAVEPOINT MIGRATION;
+
+ALTER TABLE ecu_serials ADD COLUMN sec_type TEXT NOT NULL DEFAULT "";
```

Comment:
The following comment is not exactly related to this PR and is not blocker for its merging, just simply general comment to the overall data model of aktualizr.

I think, it makes sense to have a notion of ECU in the data model (not ecu_serial) and corresponding table in the DB (ECU table), so `manifest`, `public_key`, etc are attributes of ECU not ecu_serials entity.

Reply:
Yes, I've also wondered if it should just be another table.

As we've added fields to `ecu_serials` it has effectively become what you describe and would probably more aptly named `ecus`.